src/containers/Home/HomeWidgets/CurrencyWidget.js

```diff
@@ -122,45 +123,77 @@ const CurrencyWidget = props => {
               {allSubwallets[coinObj.id] ? allSubwallets[coinObj.id].length : 1}
             </Paragraph>
             <SubWalletsLogo />
-            {
-              coinObj.mapped_to != null && (
-                <MaterialCommunityIcons
-                  name={'link'}
-                  color={Colors.secondaryColor}
-                  size={16}
-                />
-              )
-            }
+            {coinObj.mapped_to != null && (
+              <MaterialCommunityIcons
+                name={'link'}
+                color={Colors.secondaryColor}
+                size={16}
+              />
+            )}
           </View>
         </View>
-        <Paragraph
-          numberOfLines={1}
-          style={{fontSize: 16, paddingTop: 8, fontWeight: '500'}}>
-          {coinObj.testnet || uniValueDisplay === '-'
-            ? `${
-                currencyBalance == null
-                  ? '-'
-                  : normalizeNum(Number(currencyBalance), 8)[3]
-              } ${coinObj.display_ticker}`
-            : uniValueDisplay}
-        </Paragraph>
-        <Paragraph style={{fontSize: 12}}>
-          {coinObj.testnet && coinObj.proto === 'erc20'
-            ? 'Testnet ERC20 Token'
-            : coinObj.testnet
-            ? 'Testnet Currency'
-            : (coinObj.pbaas_options && !coinObj.compatible_channels.includes(GENERAL))
-            ? 'PBaaS Currency'
-            : uniValueDisplay === '-'
-            ? coinObj.proto === 'erc20'
-              ? coinObj.unlisted ? 'Unlisted Token' : 'ERC20 Token'
-              : uniValueDisplay
-            : `${
-                currencyBalance == null
-                  ? '-'
-                  : normalizeNum(Number(currencyBalance), 8)[3]
-              } ${coinObj.display_ticker}`}
-        </Paragraph>
+
+        {!showBalance ? (
+          <View>
+            <Paragraph
+             numberOfLines={1}
+             style={{fontSize: 16, paddingTop: 8, fontWeight: '500'}}
+            >********</Paragraph>
+
+            <Text
```

Comment:
Thanks, this is very close, if you just change the use of <Text/> here and also where the white space is on line 159 to <Paragraph/> and remove "color: Colors.secondaryColor", it should default to text that is a little off-white (to match the text that appears when the balances are being shown), with the same spacing as normal. This should exhibit the desired behavior of keeping all the formatting/color the same while changing only the balance to the "*****".